code/modules/dwarven/dwarven_items.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/book_of_babel/dwarven_guide
+	name = "Brokering 101: Dwarven guide"
+	desc = "An ancient tome written dwarven"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "book1"
+	w_class = 2
+
+/obj/item/book_of_babel/dwarven_guide/attack_self(mob/living/carbon/human/user)
+	if(!user.has_language(/datum/language/dwarven))
+		return FALSE
+	. = ..()
+
+/obj/item/twohanded/war_hammer
+	name = "dwarven warhammer"
+	desc = "A very heavy warhammer, used to dent skulls unless they are already dented."
```

Comment:
i get the joke and it's good but this is written in a really weird way so the joke kinda falls flat